pkg/deploy/util/util.go

```diff
@@ -72,6 +72,17 @@ func LabelForDeploymentConfig(config *deployapi.DeploymentConfig) string {
 	return fmt.Sprintf("%s/%s:%d", config.Namespace, config.Name, config.LatestVersion)
 }
 
+// ConfigSelector constructs a label selector for the provided DeploymentConfig
+func ConfigSelector(config *deployapi.DeploymentConfig) labels.Selector {
+	label := config.Template.ControllerTemplate.Selector
+	if len(label) == 0 {
+		label = map[string]string{
+			deployapi.DeploymentConfigLabel: config.Name,
```

Comment:
Oh yea, I can see it in annotations.. Not sure how I can query based on annotations though